modules/carto/package.json

```diff
@@ -46,7 +46,7 @@
     "h3-js": "^3.7.0",
```

Comment:
Have we considered bumping here to major 4.1.x ?. It looks like it manages correctly the previous error when using h3-js from web workers: https://github.com/uber/h3-js/releases